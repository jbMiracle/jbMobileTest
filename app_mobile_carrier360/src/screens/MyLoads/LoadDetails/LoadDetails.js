import React, { Component } from 'react';
import { View, Text } from 'react-native';

class LoadDetails extends Component {
  render() {
    const { params } = this.props.navigation.state;
    return (
      <View testID='load_details' style={ { flex: 1, justifyContent: 'center', alignItems: 'center' } }>
        <Text>Load Details</Text>
        <Text style={ { fontWeight: 'bold' } } >{ params && params.msg }</Text>
      </View>
    );
  }
}

export default LoadDetails;
