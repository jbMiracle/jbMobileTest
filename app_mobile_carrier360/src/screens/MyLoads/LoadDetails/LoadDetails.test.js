import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import LoadDetails from '../LoadDetails/LoadDetails';

it('can render an empty component', () => {
  const renderer = new ShallowRenderer();
  const props = {
    navigation: {
      state: {
        params: {
          msg: 'test',
        },
      },
    },
  };
  renderer.render(<LoadDetails { ...props } />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
