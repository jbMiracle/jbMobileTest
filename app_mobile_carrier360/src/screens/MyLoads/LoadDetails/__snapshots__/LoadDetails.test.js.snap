// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render an empty component 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
  testID="load_details"
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Load Details
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    test
  </Text>
</View>
`;
